README.md

```diff
@@ -201,7 +214,9 @@ accessLog:
 * `metrics.address`: Listening address of the Prometheus metrics endpoint.
 
 **Default**: :2112
-  
+
+## AWS 
```

Comment:
unintentional? or did you intend to fill out this section?

Reply:
Oops, just stray. I'll fix it.